.pylintrc

```diff
@@ -16,7 +16,7 @@
 # W1201, W1202 disable log format warning. False positives (I think)
 # W0707 disable raise-missing-from which we cant use because py2 back compat
 
-disable=C,R,duplicate-code,W0511,W1201,W1202,W0707,no-init,broad-except,bare-except
+disable=C,R,duplicate-code,W0511,W1201,W1202,W0707,no-init,broad-except,bare-except,typecheck
```

Comment:
What's the story with this?

Reply:
Looks like I accidentally removed `typecheck` in a merge a few days ago. I'd been removing it during local dev because it generates annoying false positives on Python 3.9. So adding it here would just be restoring master to it's former state.

That said I also removed it in the larger config overhaul in [my pylint PR](https://github.com/dagster-io/dagster/pull/6517), because I confirmed that it serves no purpose-- the `pylint` typecheck checker functionality is a subset of `mypy` and it also has bugs/is worse maintained.

Reply:
Got it - makes sense